### DISCUSSION TITLE
Please add sdxl lightning version

### DISCUSSION THREAD

**Discussion by rafiislambd** [rupeshs/fastsdcpu#150] [Ideas]
Please add support sdxl lightning version 4 step ..thanks

**Comment by rupeshs** [comment]
@rafiislambd  planning to add it

**Comment by rupeshs** [comment]
@rafiislambd Added SDXL ligtning support in LCM and LCM-OpenVINO mode. Please checkout release :https://github.com/rupeshs/fastsdcpu/releases/tag/v1.0.0-beta.26
